### DISCUSSION TITLE
求票

### DISCUSSION THREAD

**Discussion by mjy66** [mikumifa/biliTickerBuy#214] [聊天专区]
求一张7.13号的票，除了bml联动其他都可以

**Comment by mjy66** [comment]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173073274/c78415d6-49a3-4828-a2cc-4914a3013bb9)
还能强盗吗？各位大佬看看配置有没有问题

**Comment by JieKBK** [comment]
强实名的票怕是转让不了
